components/network/vpnc-scripts/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.illumos.org/license/CDDL.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright 2016 Jim Klimov
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		vpnc-scripts
+# Commit timestamp and previous known release (none here)
+COMPONENT_VERSION=	0.0.20160828
+COMPONENT_VERSION_TAG=	6e04e0bbb66c0bf0ae055c0f4e58bea81dbb5c3c
+COMPONENT_VERSION_TAGSHORT=	6e04e0b
+COMPONENT_ARCHIVE=	$(COMPONENT_VERSION_TAG).tar.gz
+COMPONENT_FMRI=		network/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION=	Applications/Internet
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION_TAGSHORT)
+COMPONENT_PROJECT_URL=	http://www.infradead.org/openconnect/vpnc-script.html
+COMPONENT_ARCHIVE_HASH=	\
+	sha256:988b0cbc469bdd5cb2a5af3048781e6d3b8dcd89fb9b03798ab8365d34717246
+COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)/snapshot/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE=		GPLv2
+COMPONENT_LICENSE_FILE= COPYING
+COMPONENT_SUMMARY=	Connect scripts for use with vpnc and openconnect (and similar clients)
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/justmake.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+# This is an archive with scripts, nothing to build
+build: $(SOURCE_DIR)/.prep
+	$(MKDIR) $(BUILD_DIR)
+
+# By convention (consumer defaults), this lands under /etc/vpnc/
+# The other scripts (vpnc-script-ptrtd and vpnc-script-ssh) and
+# the netunshare.c program are at the moment extremely linux
+# oriented so we don't even package them.
+install: $(SOURCE_DIR)/.prep
```

Comment:
Another option is just to add empty build and install target and to select necessary files in p5m manifest